library/src/nvcc_detail/hipsparse.cpp

```diff
@@ -9293,6 +9304,100 @@ hipsparseStatus_t hipsparseSpMV(hipsparseHandle_t           handle,
                                                      externalBuffer));
 }
 
+hipsparseStatus_t hipsparseSpGEMM_createDescr(hipsparseSpGEMMDescr_t* descr)
+{
+    return hipCUSPARSEStatusToHIPStatus(cusparseSpGEMM_createDescr((cusparseSpGEMMDescr_t*)descr));
+}
+
+hipsparseStatus_t hipsparseSpGEMM_destroyDescr(hipsparseSpGEMMDescr_t descr)
+{
+    return hipCUSPARSEStatusToHIPStatus(cusparseSpGEMM_destroyDescr((cusparseSpGEMMDescr_t)descr));
+}
+
+hipsparseStatus_t hipsparseSpGEMM_workEstimation(hipsparseHandle_t      handle,
+                                                 hipsparseOperation_t   opA,
+                                                 hipsparseOperation_t   opB,
+                                                 const void*            alpha,
+                                                 hipsparseSpMatDescr_t  matA,
+                                                 hipsparseSpMatDescr_t  matB,
+                                                 const void*            beta,
+                                                 hipsparseSpMatDescr_t  matC,
+                                                 hipDataType            computeType,
+                                                 hipsparseSpGEMMAlg_t   alg,
+                                                 hipsparseSpGEMMDescr_t spgemmDescr,
+                                                 size_t*                bufferSize1,
+                                                 void*                  externalBuffer1)
+{
+    return hipCUSPARSEStatusToHIPStatus(
+        cusparseSpGEMM_workEstimation((cusparseHandle_t)handle,
+                                      hipOperationToCudaOperation(opA),
+                                      hipOperationToCudaOperation(opB),
+                                      alpha,
+                                      (cusparseSpMatDescr_t)matA,
+                                      (cusparseSpMatDescr_t)matB,
+                                      beta,
+                                      (cusparseSpMatDescr_t)matC,
+                                      computeType,
```

Comment:
Probably should use hipDataTypeToCudaDataType(computeType) in case the hipDataType and cudaDataType dont match in future.